tests/integration/datapackage_test.py

```diff
@@ -76,8 +76,8 @@ def _create_schema(instant=True, axes=None):
             pd.read_csv(
                 io.StringIO(
                     "c_id,entity_id,filing_name,start_date,end_date,dimension_one_axis,column_one,column_two\n"
-                    'cid_1,EID1,filing,2021-01-01,2021-12-31,Total,"value 1","value 2"\n'
-                    'cid_4,EID1,filing,2020-01-01,2020-12-31,Total,"value 3","value 4"\n'
+                    'cid_1,EID1,filing,2021-01-01,2021-12-31,total,"value 1","value 2"\n'
```

Comment:
These should fix the tests that are failing on `main` right now.

Reply:
I think it was probably my diff that kept the capital T in when I reformatted the test data.